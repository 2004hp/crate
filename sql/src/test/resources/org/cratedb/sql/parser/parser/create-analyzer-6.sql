create analyzer a6 with (
    tokenizer tok (
        parameter=?,
        valuenodelist=['a', 2, 1.234],
        literal='abc'
    )
)