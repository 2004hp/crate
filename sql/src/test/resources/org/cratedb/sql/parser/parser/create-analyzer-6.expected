org.cratedb.sql.parser.parser.CreateAnalyzerNode@120b0058
name: a6
statementType: CREATE ANALYZER
	
	org.cratedb.sql.parser.parser.AnalyzerElements@10439aa9

	Tokenizer: 		
		org.cratedb.sql.parser.parser.NamedNodeWithOptionalProperties@465ed596
		tok
		Properties: 			
			org.cratedb.sql.parser.parser.GenericProperties@7c188ef1
			{parameter=number: 0
			type: null
			, valuenodelist=, literal=value: abc
			type: CHAR(3) NOT NULL
			}
	Properties: 		
		org.cratedb.sql.parser.parser.GenericProperties@3050afdd
		{}
	TokenFilters: 		
		org.cratedb.sql.parser.parser.TokenFilterList@149eef3

	CharFilters: 		
		org.cratedb.sql.parser.parser.CharFilterList@1ce6942e
